packages/CalDAV-Core.package/CALDeadlinesEventPanel.class/instance/weekTitle.st

```diff
@@ -1,4 +1,4 @@
 layouting
 weekTitle
 
-	^ 'Week ', self weekOfYear
\ No newline at end of file
+	^ 'Week ', (self index asNumber + 1)
```

Comment:
Why did you change it to "index ..."? It's a bit hard to understand, maybe use a variable?